package.json

```diff
@@ -10,9 +10,9 @@
     "benchmark": "npm run build && node -r source-map-support/register benchmarks/internal/bench.js",
     "benchmark:debug": "npm run build && node -r source-map-support/register --inspect benchmarks/internal/bench.js",
     "benchmark:prod": "npm run build:prod && node -r source-map-support/register benchmarks/internal/bench.js",
-    "test": "npm run build && nyc mocha --parallel 'tests/**/*.test.js'",
-    "test:debug": "npm run build && mocha --inspect 'tests/**/*.test.js'",
-    "test:prod": "npm run build:prod && nyc mocha --parallel 'tests/**/*.test.js'",
+    "test": "npm run build && nyc mocha 'tests/**/*.test.js'",
```

Comment:
Is parallel execution removed to the usage of the new reporter ? Which are the benefits of the new reporter ? 
Thanks in advance jerome

Reply:
It's not removed, it's enabled by default, expect for the debug script. The reporter just generate a user friendly report if needed, the report on the console remains.